Examples/SnapinsSDKKotlinExample/app/build.gradle

```diff
@@ -10,6 +10,15 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
+    signingConfigs {
+        defaultSigningConfig {
+            // FIXME publish a public default signing configuration?
```

Comment:
Let's work together to address this. It will need to be fixed before this can be merged.